New data refresh

daniel-tabarcea opened the pull request:
- added new sidebar element for dexie data refresh
- connects unicef/etools-issues#250

adi130987 commented on pmp/app-elements/data-refresh-dialog/data-refresh-dialog.html:
```diff
@@ -0,0 +1,132 @@
+            disable-confirm-btn="[[isRefreshDisabled]]"
+            on-close="_handleDialogClosed">
+      <div id="content-box">
+        <paper-dropdown-menu label="Select the data you would like to refresh">
```
since we use a dialog to display data refresh options it would be nice to show all this options using checkboxes (etools-checkable-input) and not use a dropdown.

adi130987 commented:
* remove the old refresh button

daniel-tabarcea commented:
Changed have been implemented. I also checked the mentioned promise and although it worked, I've updated it to cover the case in witch there is a partial failure.

As for the refresh button in the header, it depends on unicef-polymer/etools-page-refresh/#3
